ale_linters/go/golint.vim

```diff
@@ -0,0 +1,45 @@
+" Author: neersighted <bjorn@neersighted.com>
+" Description: golint for Go files
+
+if exists('g:loaded_ale_linters_go_golint')
+    finish
+endif
+
+let g:loaded_ale_linters_go_golint = 1
+
+function! ale_linters#go#golint#Handle(buffer, lines)
```

Comment:
I think the two handlers are the same, aren't, they" Maybe both could use a new generic `HandleUnixFormat` function in the handlers.vim file.